test/unit/SupSub.test.js

```diff
@@ -158,10 +158,10 @@ suite('SupSub', function() {
       assert.equal(mq.latex(), 'x_a^b');
 
       mq.keystroke('Left Backspace');
-      assert.equal(mq.latex(), 'xb_a');
+      assert.equal(mq.latex(), 'x_ab');
```

Comment:
I'm going to go ahead and merge this because the new behavior seems better than the old behavior. But I think we might want to revisit what happens when you delete from the left side of non-empty blocks in general. E.g.

`1+\frac{|1}{2} -> 1+|12`

also seems weird and not very useful to me.